charts/opentelemetry-demo/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.11.1
+version: 0.12.0
```

Comment:
Does this warrant a minor version bump?

Reply:
We usually bump minor chart version for minor version upgrades in images. Patch version bumps are usually made for fixes and minor changes in values.yaml interface. But we could reconsider that practice

Reply:
Id vote a minor bump for this one since it was a minor bump for the demo itself.